blosc/frame.c

```diff
@@ -995,6 +998,12 @@ uint8_t* get_coffsets(blosc2_frame_s *frame, int32_t header_len, int64_t cbytes,
         BLOSC_TRACE_ERROR("Cannot read the cbytes outside of frame boundary.");
         return NULL;
       }
+      if (chunk_nbytes != nchunks * sizeof(int64_t)) {
```

Comment:
Is it necessary? (it changes the function signature)

Reply:
This is a good consistency check, that helps to find malformed meta-info.  So it should stay.